src/Blockcore.Explorer/ClientApp/src/app/explorer/transaction/transaction.component.html

```diff
@@ -68,6 +68,23 @@ <h3><i class="fas fa-receipt"></i>&nbsp;&nbsp;Transaction Details</h3>
          <span>{{transaction.version}}</span>
       </div>
 
+      <div *ngIf="detailsVisible">
+         <span>Transaction fee</span>
+         <span>{{transaction.fee | amount}}</span>
+      </div>
+      <div *ngIf="detailsVisible">
+         <span>Size</span>
+         <span>{{transaction.size}} B</span>
```

Comment:
I think we already have an | size pipe that could be used here?

Reply:
But are they on bytes or kbytes?

Reply:
The pipe will transform it to kB yes, and in the future we could extend the pipe to display bytes or kB or MB, etc? More consistent to show 2,101kB than 2101 B, since other sizes elsewhere in the explorer uses the pipe.

Reply:
The pipe actually divides by 1024, so not sure if it will be correct.